app/src/main/java/de/westnordost/streetcomplete/quests/lanes/AddLanes.kt

```diff
@@ -0,0 +1,72 @@
+package de.westnordost.streetcomplete.quests.lanes
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder
+import de.westnordost.streetcomplete.data.osm.osmquest.OsmFilterQuestType
+
+class AddLanes : OsmFilterQuestType<LanesAnswer>() {
+
+    override val elementFilter = """
+        ways with
+          highway ~ ${ROADS_WITH_LANES.joinToString("|")}
+          and (!lanes or lanes !~ [1-9][0-9]*)
```

Comment:
What about cases with known bicycle lanes (bicycle lanes are not counted in `lanes`)? Maybe skip them?

See say https://taginfo.openstreetmap.org/tags/cycleway=lane#combinations - about 130 000 of `cycleway=lane` have no `lanes` tag and mistake here by user is quite likely.

Reply:
Yes, bicycle lanes are not counted in lanes. The quest also asks for "lanes for cars".

I don't understand your concern. Skip what? Roads with cycle lanes? Why?